cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/ArgumentsBinding.java

```diff
@@ -86,7 +87,14 @@ private void bindArgument(
 
                 Serializable expression = rawValue == null ? null : rawValue.get();
 
-                inputValue = tryEvaluateExpression(expression, evaluationContextHolder).orElse(rawValue);
+                Optional<Value> value = tryEvaluateExpression(expression, evaluationContextHolder);
+                if (value.equals(Optional.empty())) {
```

Comment:
`!value.isPresent()`

Reply:
also change the order to `value.isPresent() -> setIsExpression(true) else other`

Reply:
isPresent checks for null. I needed to check if value is empty which is a new Optional object